docs/catalog-api-use.md

```
@@ -0,0 +1,295 @@
+  POST /cloudapi/1.0.0/catalogs
+
+  Request Body:
+  {
+    "name": "New Catalog",
+    "description": "Description of the catalog",
+    "orgId": "urn:vcloud:org:87654321-4321-4321-4321-210987654321",
+    "isPublished": false
+  }
+
+  Required Fields:
+  - name: Catalog name
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)